src/test/java/net/sf/jabref/exporter/MSBibExportFormatTestFiles.java

```
@@ -0,0 +1,104 @@
+    public static Collection<String> fileNames() throws IOException {
+        List<String> files = new ArrayList<>();
+        try (Stream<Path> stream = Files.list(Paths.get(PATH_TO_FILE))) {
+            files = stream.map(n -> n.getFileName().toString()).filter(n -> n.endsWith(".bib"))
+                    .filter(n -> n.startsWith("MsBibExportFormat")).collect(Collectors.toList());
+        }
+        return files;
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        databaseContext = new BibDatabaseContext(new BibDatabase(), new MetaData());
+        charset = Charsets.UTF_8;
+        msBibExportFormat = new MSBibExportFormat();
+    }
+
+    @Test
+    public final void testPerformExport() throws IOException {
+        xmlFileName = filename.replace(".bib", ".xml");
+        tempFile = testFolder.newFile();
+        String tempFilename = tempFile.getCanonicalPath();
+        try (InputStream bibIn = MSBibExportFormat.class.getResourceAsStream(filename)) {
+            entries = testImporter.importEntries(bibIn, new OutputPrinterToNull());
+        }
+        assertNotNull(entries);
+        msBibExportFormat.performExport(databaseContext, tempFilename, charset, entries);
+        try (FileInputStream stream = new FileInputStream(tempFile)) {
+            DocumentBuilder documentBuilder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+            Document exportedDocument = documentBuilder.parse(stream, tempFilename);
+            exportedDocument.normalizeDocument();
+            Document expectedDocument = documentBuilder.parse(new File(PATH_TO_FILE + xmlFileName));
+            expectedDocument.normalizeDocument();
+            assertTrue(expectedDocument.isEqualNode(exportedDocument));
+        } catch (ParserConfigurationException | SAXException | TransformerFactoryConfigurationError e) {
+            System.out.println(e.getMessage());
```

please just throw the exceptions